apps/condo/domains/ticket/schema/TicketFile.js

```diff
@@ -43,6 +43,12 @@ const TicketFile = new GQLListSchema('TicketFile', {
                 await Adapter.delete(existingItem.file)
             }
         },
+        resolveInput: async ({ resolvedData }) => {
+            if (resolvedData['file']) {
+                resolvedData['file'].originalFilename = Buffer.from(resolvedData['file'].originalFilename, 'latin1').toString('utf-8')
```

Comment:
First of all you should check the original encoding

Reply:
i checked the incoming endocing using chardet and it can be different (ascii, utf8 etc), but when i tried converting it to buffer and specifying the correct encoding the result was incomprehensible.
then i tried converting different encodings specifying 'latin1' and it worked every time. i checked it on ascii, unicode, latin1, 7bit

Reply:
it looks suspicious